en/4-empower-staff-deliver-better-services.md

```diff
@@ -32,29 +32,25 @@ altLangPage: 4-permettre-personnel-offrir-meilleurs-services
 
 <section class="dpgn-section-guideline">
 
-## 4.1 Provide ongoing training and learning opportunities to continually improve the skills of the team and the broader network
+## 4.1 Build a team culture that is collaborative and adaptive, and is committed to enabling quality services for users
```

Comment:
The titles for 4.1, 4.2, 4.3 and 4.4 can't be changed here as they come directly from the Digital Standards. They would need to be changed separately in the Digital Standards first (they are the sub-bullets of Standard 4) and then those changes can be reflected in the Playbook. The Digital Standards are currently pending approval so changes won't be possible until after they are approved.